kitsuyui/cachepot
# enhance ci

- Install pyupgrade
- Rewrite tests for pytest
- format
- Use makefile as script runner
- use setuptools-scm to use git tag version
- Use cache and check Python version
- Support Python 3.11
